### PR TITLE
Regina/user login

### PR DESCRIPTION
**Pull request by reginavlee**
- Bug fix to add user to the DB if the user doesn't already exist.
- Updated lock widget to only show facebook login option until other services are implemented.

### CODE REVIEW

**Comment by reginavlee** [comment]
@armenr @andrewjyi pls review and merge into master, and then also redeploy!

File: app/src/actions/auth.js [hunk 1]
```diff
@@ -20,11 +17,30 @@ export function checkLogin() {
...
+          email: profile.email,
+          fbID: userID,
+        }
+        console.log('new user', newUser)
+        axios
+          .get(`http://localhost:3000/users?authID=${userID}`)
+          .then((response) => {
+            console.log('response in axios get request', response)
+            if (!response.data.length) {
+              axios.post('http://localhost:3000/users', newUser)
```
Comment (armenr) [inline app/src/actions/auth.js]: We've hard-coded here. 

Let's merge all these together, then open a new PR for refactoring ALL references to the API (heroku OR localhost) into a variable that's loaded via .env

File: app/src/containers/HomePage/HomePage.js [hunk 2]
```diff
@@ -36,6 +28,7 @@ class HomePage extends Component {
     if (this.props.isAuthenticated) {
       let fbID = this.props.profile.user_id
       let id = fbID.split('|')
+      console.log('id', id)
```
Comment (armenr) [inline app/src/containers/HomePage/HomePage.js]: Let's utilize the forthcoming refactor PR to also factor out all references to console.log

File: app/src/containers/NavBarContainer/NavBarContainer.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      dispatch(logoutSuccess())
+      hashHistory.push('/')
+      location.reload()
+    }
+  }
+}
+
+const NavBarContainer = connect(mapStateToProps, mapDispatchToProps)(NavBar)
+
+export default NavBarContainer
```
Comment (armenr) [inline app/src/containers/NavBarContainer/NavBarContainer.js]: Need a newline @ the end of this file.

**Pull request merged**